---
layout: archive
permalink: /
title: "St√©phanie Chevalier"
excerpt: "About me"
author_profile: true
redirect_from:
  - /about/
  - /about.html
---
## Postdoctoral fellow @ Servier

<span style="color: #b45f06; font-weight: bold; text-shadow: 1px 1px 0px rgba(0,0,0,0.15);">Assist in drug combination strategies through interaction network modeling</span> and involved in the concrete needs of therapeutic solution research to **predict the behavior of a biological system when subjected to perturbations**.

<hr class="hr_gradient" />

## Motivation

<span style="color: #b45f06; font-weight: bold; text-shadow: 1px 1px 0px rgba(0,0,0,0.15);">High interest in multisystemic disorders.</span> I aim at contributing in related fundamental research and helping **the translation of basic science results into health care advances**.

<span style="color: #b45f06; font-weight: bold; text-shadow: 1px 1px 0px rgba(0,0,0,0.15);">Special affinity for the holistic approach of systems biology</span> to unravel and predict biological phenomena.
I trained in bioinformatics and had internships in different areas of this discipline: the challenge of **data integration**, **NGS analysis**, **structural analysis** for drug design, and finally the one of complex **biological systems analysis and modelling**. I then approached related issues from different viewpoints.  

<hr class="hr_gradient" />

## PhD topic

**PhD in computer science for a systems biology issue:** <important>modelling of regulatory mechanisms</important>.

I contributed to <important>a method for automatic inference of discrete dynamical models of biological interactions</important> governing complex cell behaviors, called <important>BoNesis</important>.
It allows to **model regulatory mechanisms of biological processes with complex dynamic properties**, such as cell differentiation, by taking into account **knowledge on thousands of genes**.

[![BoNesis-principle](../images/bonesis_principle.png){:class="img-page"}](https://github.com/bnediction/bonesis)

This modeling confronts **prior knowledge** on interactions with **observations** along the process to model (bulk/single cell gene expressions, perturbations, mutations...). It can then <important>enumerate all Boolean networks reproducing a complex behavior</important> (under _[mp semantics](https://hal.archives-ouvertes.fr/hal-01864693v2/document)_), e.g. cell differentiation. It can also be used to help <important>select relevant nodes among a large prior knowledge network</important> (e.g. from public interaction databases).

Method based on Answer-Set-Programming.
