---
layout: skills
title: "Skills overview"
permalink: /skills/
author_profile: true
---

* **<span class=important>Programming:</span>**  
**Python**, **C**, **C++**, **Bash**, **R**, **SQL** (PostgreSQL, mySQL)    
**Git**, **Jupyter Notebook**  
**Answer-Set-Programming**  
**LaTeX**, **markdown**, **HTML/CSS**
* **<span class=important>Machine learning:</span>**  
**Regression** (lin., polyn., logistic, GLM)  
**Classification** (decision tree, kNN)  
**Clustering** (hierarchical, k-means)  
**Ensemble methods** (random forest)  
**Dimensionality reduction** (PCA, MDS)  
**Cross-validation**
* **<span class=important>NGS analysis:</span>**
    + *<informative>tertiary:</informative>*  
    **Differential Expression analysis** (edgeR, DESeq)  
    **Gene Set Enrichment Analysis** (Metascape)  
    **Single-cell data biases**
    + *<informative>secondary:</informative>*  
    **Quality Control** (FastQC)  
    **Read cleaning** (trimmomatic, cutadapt)  
    **Mapping** (genome: STAR; transcriptome: Kallisto) or **de novo assembly** (Trinity)  
    **Gene expression quantification** (FeatureCounts)
* **<span class=important>Data analysis tools:</span>**  
**Data mining** (pandas, numpy, scipy, sklearn, networkx)  
**Visualisation** (matplotlib, seaborn, graphviz, cytoscape)
* **<span class=important>Public biological databases:</span>**  
NCBI/EMBL/DDBJ, Gene Ontology, UniProt, DoRothEA, Signor, Reactome, PDB...
* **<span class=important>Theoretical computer science:</span>**  
**Algorithmics**, **graphs**, **complexity**
* **<span class=important>Teaching skills:</span>**  
Design of teaching materials, exercises, exams
* **<span class=important>Operating Systems:</span>**  
**Linux** (Debian-based mostly), **Windows**
* **<span class=important>Languages:</span>**  
**French** (native language), **English**
