---
layout: skills
title: "Overview of my skills"
permalink: /skills/
author_profile: true
---

* **Programming:**  
Python, Bash, C, C++ --
SQL (PostgreSQL, mySQL) --
Answer-Set-Programming --
HTML/CSS, JavaScript --
LaTeX
* **Machine learning:**  
Regression (lin., polyn., logistic, GLM) --
Classification (decision tree, kNN) --
Clustering (hierarchical, k-means) --
Ensemble methods (random forest) --
Dimensionality reduction (PCA, MDS) --
Cross-validation
* **Software and tools:**  
Data handling/analysis (pandas, numpy, scipy, sklearn, networkx)  
Visualisation (matplotlib, seaborn, graphviz, cytoscape)
* **Public biological databases:**  
NCBI/EMBL/DDBJ, Gene Ontology, UniProt, DoRothEA, Signor, KEGG, OMIM, PDB...
* **Theoretical computer science:**  
Algorithmics, Graphs, Complexity, Languages
* **NGS tertiary analysis:**  
Recent experience of Gene Set Enrichment Analysis (Metascape).  
Awareness of single-cell data biases. Knowledge on normalization and differential expression analysis (edgeR, DESeq).
* **NGS secondary analysis:**  
Knowledge on quality control (FastQC), raw read cleaning (trimmomatic, cutadapt), mapping (genome: STAR; transcriptome: Kallisto) or de novo assembly (Trinity), gene expression quantification (FeatureCounts).
* **Teaching skills:**  
Design of teaching materials, exercises, exams
* **Operating Systems:**  
Linux (Debian-based mostly), Windows
* **Languages:**  
French (native language), English
